supervisely/api/file_api.py

```diff
@@ -682,11 +682,11 @@ def remove_from_agent(self, team_id: int, path: str) -> None:
 
     def remove(self, team_id: int, path: str) -> None:
         """
-        Removes file from Team Files.
+        Removes path from Team Files.
```

Comment:
if path is file - remove it
if path is directory - remove entire directory with all inclusive files recursively